changelog_unreleased/graphql/10655.md

```diff
@@ -0,0 +1,19 @@
+#### Fix missing space after keyword in anonymous graphql operations (#10655 by @patriscus)
```

Comment:
Could you please rename the file and specify the number of the PR here instead of the number of the issue?

Reply:
@thorn0 thanks for reviewing so promptly and sorry about the wrong number, changed it now